formats/vanish/notification/schema.json

```diff
@@ -0,0 +1,45 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "publishing_app"
```

Comment:
Is this right that only the publishing app is required? Is this to allow us to vanish pathless content? If so what's the identifier for the thing we want to vanish? Would specifying a `content_id` OR `base_path` be possible?